integration_test/third_party_apps_data/agent/ops-agent/linux/enable_mysql

```diff
@@ -16,7 +16,18 @@ sudo tee /etc/google-cloud-ops-agent/config.yaml > /dev/null << EOF
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+metrics:
+  receivers:
+    mysql_metrics:
+      type: mysql
+      collection_interval: 10s
+      username: root
```

Comment:
We should be testing the defaults (i.e. the unit test should not specify `username`, `password`, or `collection_interval`).

Reply:
updated

Reply:
Looks like `collection_interval` was removed in https://github.com/GoogleCloudPlatform/ops-agent/pull/218/commits/d9f213f6764b6dca1e2ea9f456c31be1a089657e (which addressed this thread) but added back in https://github.com/GoogleCloudPlatform/ops-agent/commit/ab5db88f88f20c97eeef19093f4ac089793517e1.

Any special reason behind it? @Mrod1598

Reply:
becauase it was saying it was required without it

Reply:
Looks like `collection_interval` is indeed defined as `required`. It should be optional though. Since it's not directly related to this PR, I'll file a separate bug.